src/main/java/frc/robot/Constants.java

```diff
@@ -1,19 +1,162 @@
-// Copyright (c) FIRST and other WPILib contributors.
-// Open Source Software; you can modify and/or share it under the terms of
-// the WPILib BSD license file in the root directory of this project.
-
 package frc.robot;
 
-/**
- * The Constants class provides a convenient place for teams to hold robot-wide numerical or boolean
- * constants. This class should not be used for any other purpose. All constants should be declared
- * globally (i.e. public static). Do not put anything functional in this class.
- *
- * <p>It is advised to statically import this class (or one of its inner classes) wherever the
- * constants are needed, to reduce verbosity.
- */
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.SwerveDriveKinematics;
+import edu.wpi.first.math.trajectory.TrapezoidProfile;
+import edu.wpi.first.math.util.Units;
+import frc.lib.util.COTSFalconSwerveConstants;
+import frc.lib.util.SwerveModuleConstants;
+
 public final class Constants {
```

Comment:
Also I would recommend that all constants follow the naming convention of `THIS_IS_A_CONSTANT` with all caps and spaces as underscores. This is how we have done it in the past. But this is your choice, just because we have done it in the past doesn't mean it's the correct way, but I would recommend some kind of naming convention.

Reply:
Fixed! This is how it should be, it got messed up when copied and pasted. It is the way I am used to as well and should be retained.